app/models/application.rb

```diff
@@ -18,6 +18,12 @@ class Application < ApplicationRecord
   after_create :superkey_workflow
   after_destroy :superkey_workflow
 
+  IGNORE_RAISE_EVENT_ATTRIBUTES_LIST = %w[availability_status availability_status_error].freeze
```

Comment:
Do we ever see a need to export these out to Environment variables ( potentially different for differrent envs?)  Or is a constant going to be fine

Reply:
it looks that it is enough to have those attributes just in constant as raising event `Application#update` (after updating  `availability_status` or `availability_status_error`) doesn't any consumer regardless of environments.

@slemrmartin can you confirm it, please ?  Thanks!

Reply:
Constant is enough. I don't think we'll need different values per env